openfisca_core/taxscales.py

```diff
@@ -153,6 +153,14 @@ def calc(self, base):
         return np.dot(self.amounts, a.T > 0)
 
 
+class LookupTaxScale(AmountTaxScale):
```

Comment:
Add docstring to explicit this scale behaviour?

Reply:
Done. However I have no idea how to describe the *RateTaxScale classes and leave that to future improvement :)

Reply:
A rate tax scale compute the tax as a fraction of the taxable amount (or taxable base) according to a marginal or average tax scale.